src/pages/Login/Login.js

```diff
@@ -0,0 +1,58 @@
+import React, { Component } from 'react';
+import './Login.scss';
+// import '../../styles/reset.scss';
+
+class Login extends Component {
+    render() {
+        return (
+        <div className="loginContainer">
+            <div class="logo">
+                <img class='logoImage' src='./images/logo.png' />
+            </div>  
+            <div className="messageBox">
+                <img className="gmarketLogo" src="https://pics.gmarket.co.kr/pc/single/kr/common/image__logo.png" alt="gmarketLogo" />
+                <span class='gmarketMessage'>아이디로 이용이 가능합니다.</span>
+            </div>
+            <form>
+                <div className = 'inputBox'>
```

Comment:
띄어쓰기! 수정해주세요. 전체적으로 검토 부탁드립니다.
```suggestion
                <div className='inputBox'>
```